tools/mtv-api-tests.py

```
@@ -0,0 +1,1177 @@
+    },
+    "powermax": {
+        "label": "Dell PowerMax",
+        "fields": [("powermax_symmetrix_id", "Symmetrix ID (e.g. 000123456789)")],
+    },
+    "powerstore": {"label": "Dell PowerStore", "fields": []},
+    "infinibox": {"label": "Infinidat InfiniBox", "fields": []},
+    "flashsystem": {"label": "IBM FlashSystem", "fields": []},
+}
+
+
+class RunMode(str, Enum):
```

Ruff (0.15.4)

[warning] Class RunMode inherits from both `str` and `enum.Enum`

Inherit from `enum.StrEnum`

(UP042)